flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/utils/ExpressionTestBase.scala

```diff
@@ -312,7 +311,8 @@ abstract class ExpressionTestBase {
         env.getConfig,
         Collections.emptyMap(),
         Collections.emptyMap(),
-        null)
+        null,
+        new JobID())
```

Comment:
hmmm... This is a concerning usage of the `RuntimeUDFContext`. I thought it should be used only in the `CollectionExecutor`. It is not a concern of this PR though.

@twalthr @wuchong Do you think we could replace it with something different here?